frontend/src/lang/modules/en.ts

```diff
@@ -1693,7 +1699,8 @@ const message = {
         backupRecoverMessage: 'Please enter the compression or decompression password (leave blank to not set)',
     },
     license: {
-        community: 'Free',
+        community: 'OSS',
+        oss: 'Open Source Software',
         pro: 'Pro',
         trial: 'Trial',
         office: 'Official',
```

Comment:
Some key changes observed:

1. **Removed Duplicate `per` Field**: The "per" field has been removed from the constant `message`.

2. **Added New Per-Duration Fields**:
   - `perMonthHelper`: For scheduling tasks on specific days of each month.
   - `perWeekHelper`: For scheduling tasks every seven days on specified weekdays.
   - Additional fields for per-day, hour, and n-day intervals (n hours, n minutes, n seconds).

These additions expand the capabilities of task scheduling to cover more complex time periods beyond daily, weekly, hourly, etc.

3. **Renamed License Terms**:
   - Changed "community" to "OSS".
   - Renamed "pro" to "Pro".

These name changes may be intended to clarify terminology or indicate different licensing types.

Overall, these updates enhance the flexibility and functionality of the scheduler, especially when dealing with long-term recurring schedules. They also introduce new features that align with modern IT practices (such as open-source software licenses).